DOCS/man/ipc.rst

```diff
@@ -316,6 +316,20 @@ extra commands can also be used as part of the protocol:
     lead to breakages with future mpv releases. Instead, make a feature request,
     and ask for a proper event that returns the information you need.
 
+``msg``
+    Write a log message using the client's log instance. The first parameter to
```

Comment:
Not sure if "log instance" is at all meaningful to readers of the documentation. That's more of an mpv-internal thing.